### DISCUSSION TITLE
Create Shortcut

### DISCUSSION THREAD

**Discussion by abakasam** [ungoogled-software/ungoogled-chromium-android#82] [Ideas]
How do you Create Shortcut or Pin to Shelf?

https://www.groovypost.com/howto/pin-a-website-as-an-app-to-the-chromebook-taskbar/

Sam

**Comment by wchen342** [comment]
The article you posted is about ChomeOS not Android.

**Comment by abakasam** [comment]
It works the same for Android... comes up as Add to Home Screen:

https://browserhow.com/how-to-add-to-home-screen-shortcut-links-with-chrome-android/
https://apkals.com/download/download-chromium-apk/

Need this feature in Android.

Chrome might come up as Create Shortcut, that was in 2019:

https://www.groovypost.com/howto/pin-a-website-as-an-app-to-the-chromebook-taskbar/

Sam

**Reply by wchen342** [reply]
The functionality already exists, in the three dot menu -> Add to home screen.

**Reply by abakasam** [reply]
Do not see.. Add to Home Screen

![Screenshot 2021-03-31 12 01 03 AM](https://user-images.githubusercontent.com/58202540/113092777-4607f700-91b4-11eb-9423-d775faa450fd.png)

**Reply by wchen342** [reply]
Is that the tablet UI? Probably chromium hides the button? What if you use phone UI?

**Reply by abakasam** [reply]
![Screenshot 2021-03-31 2 28 31 PM](https://user-images.githubusercontent.com/58202540/113199990-6b851700-922d-11eb-9f63-6565285dedec.png)

**Reply by wchen342** [reply]
What I was trying to figure out it whether the different layout changed the menu. You UI looks different from the usual one on phones so I think it is the UI specific for tablets (chromium will switch aotumatically if the screen is wider than a certain number).

**Reply by abakasam** [reply]
Works from APKPals on my phone not Chromebook.

From Ungoogle Chromium: https://github.com/ungoogled-software/ungoogled-chromium-android/releases/download/89.0.4389.90-1/ChromeModernPublic_x86.apk
![Screenshot 2021-04-01 8 56 59 AM](https://user-images.githubusercontent.com/58202540/113305050-42659480-92c8-11eb-8541-f87c13f1c9c8.png)

From APKPals: https://apkals.com/chromium-apk/
![Screenshot_20210401-085317](https://user-images.githubusercontent.com/58202540/113304924-1d712180-92c8-11eb-942a-0e8a35a7731f.png)
![Screenshot_20210401-085332](https://user-images.githubusercontent.com/58202540/113304978-2e219780-92c8-11eb-9a36-8bfb7280b4e5.png)

**Reply by wchen342** [reply]
Yeah so it's a chromium issue, they did this on purpose. You need to report to their bug tracker.
